modules/objc/examples/code_snippets/SampleCodeTest.m

```diff
@@ -934,6 +934,25 @@ - (void) dontTestReplicationPullFilter {
     // end::replication-pull-filter[]
 }
 
+//  Added 2/Feb/21 - Ian Bridge
+    - void dontTestCustomHeartbeat {
+    // tag::replication-set-heartbeat[]
+        id target =
+          [[CBLURLEndpoint alloc] initWithURL: [NSURL URLWithString: @"ws://foo.cbl.com/db"]];
+
+        CBLReplicatorConfiguration* config = 
+            [[CBLReplicatorConfiguration alloc] initWithDatabase: db target: target];
+        config.type = kCBLReplicatorTypePush;
+        config.continuous: YES];
```

Comment:
```suggestion
        config.continuous: YES;
```

Reply:
do you think, for consistency should we need to add comment `//  other config as required . . .` like Java and .NET?